### PR TITLE
Added support for Asp.net Identity.

### PR DESCRIPTION
**Pull request by DeradoZA**

### CODE REVIEW

File: Cobit-19/appsettings.json
Comment (GerniVisser) [inline Cobit-19/appsettings.json:1]: Does this mean that an entirely new Database is used to save the Identity stuff? The current DB is called CobitDb and the identity one is Cobit-19

File: Cobit-19/Models/AppDbContext.cs
Comment (GerniVisser) [inline Cobit-19/Models/AppDbContext.cs:1]: DbContext is an EntityFramework extension. Wil is Identity compatible with that?

**Review by GerniVisser — COMMENTED** [review]
There are just a few thing I want you to check like the appSettings file

File: Cobit-19/appsettings.json
Comment (DeradoZA) [reply Cobit-19/appsettings.json:1]: That's my bad, that connection string isn't necessary. Was just something that was generated by default which Ivé removed now.

**Review by GerniVisser — APPROVED** [review]

**Pull request merged**